kad-dht/README.md

```diff
@@ -455,6 +455,53 @@ multiaddrs are stored in the node's peerbook.
 
 ---
 
+## Practical Examples
+<details>
+  <summary>Searching the <b>IPFS</b> Kad DHT for a Peer's Public Key</summary><blockquote>
```

Comment:
I think my message on https://discuss.libp2p.io/ was misleading. Sorry for that. `/pk` is an IPFS concept and not a llibp2p concept. Thus I don't think it should belong in libp2p/specs but instead in ipfs/specs.

Now unfortunately IPFS does not (yet) have a Kademlia specification, i.e. a document that describes how IPFS uses the libp2p Kademlia protocol. This is tracked in https://github.com/ipfs/specs/issues/345. //CC @lidel

I suggest moving the `/pk` related example to ipfs/specs.

Reply:
:+1: to moving IPFS-specific things to ipfs/specs (fine to open a PR draft with copy&paste and then mention in https://github.com/ipfs/specs/issues/345 so we can take it from there).